lib/_tls_common.js

```diff
@@ -153,6 +153,19 @@ exports.createSecureContext = function createSecureContext(options) {
     }
   }
 
+  const sigalgs = options.sigalgs;
+  if (sigalgs !== undefined && sigalgs !== null) {
```

Comment:
since the type has to be string, just remove the check for ` && sigalgs !== null` please